raiden/smart_contracts/ChannelManagerLibrary.sol

```diff
@@ -50,30 +56,36 @@ library ChannelManagerLibrary {
     /// @notice newChannel(address, uint) to create a new payment channel between two parties
     /// @dev Create a new channel between two parties
     /// @param partner (address) the address of the partner
-    /// @param settleTimeout (uint) the settleTimeout in blocks
+    /// @param settle_timeout (uint) the settle timeout in blocks
     /// @return (address) the address of the NettingChannelContract.
-    function newChannel(Data storage self, address partner, uint settleTimeout) returns (address) {
-        address channelAddress;
+    function newChannel(
+        Data storage self,
+        address partner,
+        uint settle_timeout
+    )
+        returns (address)
```

Comment:
no I strongly disagree here.

Reasoning:

When one skims through the code quickly (which is how you should read code on your first pass) the eye searches for indented blocks in order to identify function bodies and quickly deduce functionality. 

Here there is a change of indentation from the closing parentheses to the `returns`.

Perhaps if you don't want it in the same line, a compromise would be to close the parentheses after `uint settle_timeout` so that at least they are all on the same level?